catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/AbstractDurableFileConsumer.java

```diff
@@ -35,7 +34,10 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(AbstractDurableFileConsumer.class);
 
-  FileSystemPersistenceProvider fileSystemPersistenceProvider;
+  //  Implement these to serializing the observers states.
```

Comment:
Is this supposed to say "Implement these to serialize the observers states."?

Reply:
Yep it should